src/configuration.ts

```diff
@@ -15,6 +16,7 @@ export type ConfidentialClientConfiguration = {
   owners: Array<string>;
   wellKnownUri: string;
   jwk: ConfidentialClientJwk;
+  proxyUrl?: string;
```

Comment:
Let's not add this to this interface, which describes the format of the credentials file